readme.md

```diff
@@ -397,6 +397,7 @@
 - [Design systems](https://github.com/klaufel/awesome-design-systems#readme) - Collection of reusable components, guided by rules that ensure consistency and speed.
 - [Inertia.js](https://github.com/innocenzi/awesome-inertiajs#readme) - Make single-page apps without building an API.
 - [MDBootstrap](https://github.com/mdbootstrap/awesome-mdbootstrap#readme) - Templates, layouts, components, and widgets to rapidly build websites.
+- [ExtJS](https://github.com/abenhamdine/awesome-extjs) - Complete Javascript framework with UI components
```

Comment:
According to the PR guidelines this should end in a dot and the link should go to the readme.

Reply:
Indeed

Reply:
I can't see a change yet, did you want to change that?